frontend/Makefile

```diff
@@ -0,0 +1,72 @@
+dev.start.admin: init dev.backend.start
+	@lerna run start:dev --scope admin-ui
+
+dev.start.consumer: init dev.backend.start
+	@lerna run start:dev --scope consumer-ui
+
+dev.start.all: init dev.backend.start
+	@lerna run start:dev
+
+init:
+	@make --directory .. bootstrap
+
+dev.end: dev.backend.end
+
+dev.backend.start:
+	@make --directory ../backend dev.start
+
+dev.backend.end:
+	@make --directory ../backend dev.end
+
+test.admin-unit:
+	@lerna run test:unit --scope admin-ui
+
+test.consumer-unit:
+	@lerna run test:unit --scope consumer-ui
+
+test.admin-integration:
+	@lerna run cypress:integration --scope admin-ui
+
+test.consumer-integration:
+	@lerna run cypress:integration --scope consumer-ui
+
+test.admin-e2e:
+	@lerna run cypress:e2e --scope admin-ui
+
+test.consumer-e2e:
+	@lerna run cypress:e2e --scope consumer-ui
```

Comment:
Those are really developer command to test ui locally. They are not intended to be used on the ci server.